### PR TITLE
Show all snippets

### PR DESCRIPTION
**Pull request by Osmose**
https://bugzilla.mozilla.org/show_bug.cgi?id=744581

Creates a page for users to view all snippets that are currently available from the snippets service.

A few notes:
- `<script>` and `<style>` tags are intentionally left in, as they often are integral to the snippet preview. In the future we might escape or remove these if any issues arise.
- There are no tests for the view; it's simple enough currently.

### CODE REVIEW

File: apps/homesnippets/templates/show_all_snippets.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                              {% else %}
+                                {% trans 'Inclusion Rule' %}
+                              {% endif %}
+                            </li>
+                            {% for field in fields %}
+                              {% with rule|getattribute:field as value %}
+                                {% if value != '' and value != None %}
+                                  <li>
+                                    {% blocktrans %}
+                                      <strong>{{ field }}</strong>: <code>{{ value }}</code>
```
Comment (pmclanahan) [inline apps/homesnippets/templates/show_all_snippets.html:56]: Will this work for translation? I thought this bit of l10n had to be handled in the DB.

File: apps/homesnippets/templatetags/getattribute.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django import template
+from django.conf import settings
+
+
+register = template.Library()
+
+
+def getattribute(value, arg):
+        """Gets an attribute of an object dynamically from a string name"""
+
```
Comment (pmclanahan) [inline apps/homesnippets/templatetags/getattribute.py]: NIT: superfluous blank line

**Comment by pmclanahan** [comment]
r+ as long as the l10n question is answered.

File: apps/homesnippets/templates/show_all_snippets.html (hunk 1 above)
Comment (Osmose) [inline apps/homesnippets/templates/show_all_snippets.html:56]: It'll work; translators will just see placeholders for field and value.

Comment (pmclanahan) [inline apps/homesnippets/templates/show_all_snippets.html:56]: Then I don't see the point of the blocktrans block. There don't seem to be any strings in there.

Comment (Osmose) [inline apps/homesnippets/templates/show_all_snippets.html:56]: Right-to-left languages, mainly. Not sure if there's an alternate way of handling that then reversing the order in the translation string though.

**Pull request merged**